api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -791,4 +792,43 @@ public void getCountOfDrugs_shouldReturnTheTotalNumberOfMatchingNumbers() {
 		assertEquals(new Integer(1),
 		    conceptService.getCountOfDrugs(phrase, conceptService.getConcept(conceptId), true, true, true));
 	}
+
+	/**
+	 * @see ConceptServiceImpl#saveConceptProposal(ConceptProposal) 
+	 */
+	@Test
+	public void saveConceptProposal_shouldReturnTheSavedConceptProposalObject() {
+		ConceptProposal conceptProposal = new ConceptProposal();
+		conceptProposal.setOriginalText("OriginalText");
+		List<ConceptProposal> existingConceptProposals = conceptService.getConceptProposals("OriginalText");
+		assertTrue(existingConceptProposals == null || existingConceptProposals.isEmpty());
+		ConceptProposal savedConceptProposal = conceptService.saveConceptProposal(conceptProposal);
+		assertEquals(conceptProposal, savedConceptProposal);
+	}
+
+	/**
+	 * @see ConceptServiceImpl#saveConceptProposal(ConceptProposal)
+	 */
+	@Test
+	public void saveConceptProposal_shouldReturnTheUpdatedConceptProposalObject() {
+		ConceptProposal conceptProposal = new ConceptProposal();
+		conceptProposal.setOriginalText("OriginalText");
+		ConceptProposal savedConceptProposal = conceptService.saveConceptProposal(conceptProposal);
+		savedConceptProposal.setOriginalText("AnotherOriginalText");
+		ConceptProposal updatedConceptProposal = conceptService.saveConceptProposal(savedConceptProposal);
+		assertEquals("AnotherOriginalText", updatedConceptProposal.getOriginalText());
```

Comment:
This test case is not required. You can remove it

Reply:
@PermissionError did you also look at  OpenMRS Unti Test Naming Conventions at https://wiki.openmrs.org/display/docs/Unit+Testing+Conventions#UnitTestingConventions-NamingTestMethods?

Reply:
My apologies, did not know that the convention existed. The test methods have been renamed to be more concise. Thanks for the pointers!